MIAUI/MIAUI/ViewModels/ProfilePageViewModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace MIAUI.ViewModels;
+public partial class ProfilePageViewModel
+{
+    // Display info here
+
+
+
+}
```

Comment:
Let's delete this file since it's not being used anywhere! Make sure also to delete any references to it

Reply:
This is already deleted on the MIAUI PR, here I am only changing anything relevant to Azure AD B2C

Reply:
Please remove this file from this PR since this PR shouldn't have any MIAUI things in it.